flocker/provision/_rackspace.py

```diff
@@ -15,6 +16,14 @@ def provision_rackspace(node, package_source, distribution):
     """
     Provision flocker on this node.
     """
+    if distribution in ('centos-7',):
+        run(
+            username='root',
+            address=node.address,
+            commands=task_upgrade_kernel_centos(),
```

Comment:
This was just an oversight for AWS, and digitialocean support hasn't landed yet.